src/patterns/validation/index.md.njk

```diff
@@ -67,11 +67,13 @@ There are 2 ways to carry out validation:
 + server side validation
 + client side validation (in the browser, using either JavaScript or HTML)
 
-You’ll always need to carry out server side validation, because there’s no guarantee that client side validation will work in all circumstances. For example users can bypass client side validation, or [JavaScript can fail to load](https://www.gov.uk/service-manual/technology/using-progressive-enhancement).
+You’ll always need to carry out server side validation.
```

Comment:
```suggestion
You’ll always need to carry out server side validation, even if you use client side validation. This is because there’s no guarantee that client side validation will work in all circumstances. For example users can bypass client side validation, or [JavaScript can fail to load](https://www.gov.uk/service-manual/technology/using-progressive-enhancement).```

Reply:
Yeah, thanks @EoinShaughnessy, this looks much better than my suggestion! I think this is fine.